src/GraphQL.Tests/Utilities/SchemaBuilderTestBase.cs

```diff
@@ -30,14 +29,16 @@ public ExecutionResult AssertQuery(Action<ExecuteConfig> configure)
 
             var queryResult = CreateQueryResult(config.ExpectedResult);
 
-            return AssertQuery(_ =>
-            {
-                _.Schema = schema;
-                _.Query = config.Query;
-                _.Inputs = config.Variables.ToInputs();
-                _.Root = config.Root;
-                _.ThrowOnUnhandledException = config.ThrowOnUnhandledException;
-            }, queryResult);
+            return AssertQuery(
+                _ =>
+                {
+                    _.Schema = schema;
+                    _.Query = config.Query;
+                    _.Inputs = SystemTextJson.StringExtensions.ToInputs(config.Variables);
```

Comment:
```suggestion
                    _.Inputs = config.Variables.ToInputs();
```

Reply:
Ah yea, was getting a method not found at one stage and trying to figure out why. Eventually it disappeared once I deleted all my bin and obj folders. VS is fun sometimes.